lib/components/form/connected-location-field.tsx

```diff
@@ -23,6 +26,31 @@ const ConnectedLocationField = connectLocationField(StyledLocationField, {
   includeLocation: true
 })
 
+export function GeocodedOptionIcon({
+  feature = {}
+}: {
+  feature: {
+    properties?: { modes?: string[]; source: string }
+  }
+}): React.ReactElement {
+  // FIXME: add types to context
+  // @ts-expect-error No type on ComponentContext
+  const { ModeIcon } = useContext(ComponentContext)
+
+  const { properties } = feature
+  if (feature && properties) {
+    const { modes } = properties
+    if (modes && modes.length > 0) {
+      return (
+        <IconWrapper>
+          <ModeIcon mode={modes[0].toLowerCase()} role="img" />
```

Comment:
If you've got a `role="img"` on here, I think you'll need an `aria-label` too! If they're decorative, you can probably just `aria-hidden`

Reply:
Amazing catch thank you!